src/discord.h

```diff
@@ -197,9 +196,6 @@ class Discord : public QObject
     QPair<int64_t ,int64_t> getTimeStamps(Host* pHost) const { return qMakePair(mStartTimes.value(pHost), mEndTimes.value(pHost)); }
     QPair<int, int> getParty(Host* pHost) const { return qMakePair(mPartySize.value(pHost), mPartyMax.value(pHost)); }
 
-    // Returns the Discord user received from the Discord_Ready callback
-    QStringList getDiscordUserDetails() const;
```

Comment:
Though this is not currently used it was defined for the future and was useful during debugging. Unless there is a compelling reason to remove it it may be useful for the Lua API and I would not like to see it removed without reason.

Reply:
OK, will restore this.